go/vt/vtgate/tabletgateway.go

```diff
@@ -62,6 +70,9 @@ func init() {
 		fs.StringVar(&CellsToWatch, "cells_to_watch", "", "comma-separated list of cells for watching tablets")
 		fs.DurationVar(&initialTabletTimeout, "gateway_initial_tablet_timeout", 30*time.Second, "At startup, the tabletGateway will wait up to this duration to get at least one tablet per keyspace/shard/tablet type")
 		fs.IntVar(&retryCount, "retry-count", 2, "retry count")
+		fs.BoolVar(&balancerEnabled, "balancer_enabled", false, "Whether to enable the tablet balancer to evenly spread query load")
+		fs.StringSliceVar(&balancerVtgateCells, "balancer_vtgate_cells", []string{}, "When in balanced mode, a comma-separated list of cells that contain vtgates (required)")
```

Comment:
Everything else is cosmetic, but this seems important.
What happens if you add vtgates to a cell that didn't previously have any? You would have to restart every vtgate with a new value for this flag?
It may not be very common in practice, but it should be documented somewhere.
Of course, the real answer is that vtgates should be given a list of all cells and discover which ones contain vtgates, but we've so far avoided adding vtgates to the topo.

Reply:
>What happens if you add vtgates to a cell that didn't previously have any? You would have to restart every vtgate with a new value for this flag?

Yep. For the current algorithm to work as designed, it's required that all the vtgates know the fully set of cells that are running vtgates and that the load among these cells is roughly even. This allows each individual vtgate to make the assumption about what the others will do without needing signaling.

If you want to change this assumption therefore, you do have to restart the whole fleet to take it into account, just like any other vtgate flag change.

> It may not be very common in practice, but it should be documented somewhere.

That makes sense. I'll include something to this effect when we write the website docs.

Reply:
> Of course, the real answer is that vtgates should be given a list of all cells and discover which ones contain vtgates, but we've so far avoided adding vtgates to the topo.

Yeah - that's obviously beyond the scope of this PR, and it would in theory reduce the dependency that query inflows per cell need to be balanced (i.e. you could use the number of gates as a proxy for how much load you expect).